.pyinstaller/hooks/hook-sunpy.py

```diff
@@ -0,0 +1,25 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2013-2019, PyInstaller Development Team.
+#
+# Distributed under the terms of the GNU General Public License with exception
+# for distributing bootloader.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#-----------------------------------------------------------------------------
+
+from PyInstaller.utils.hooks import collect_data_files, collect_submodules, copy_metadata, collect_entry_point
+
+
+datas, hiddenimports = collect_entry_point("pytest11")
+
+datas += collect_data_files("sunpy")
+datas += collect_data_files("dask")
+datas += collect_data_files("drms")
+datas += copy_metadata("sunpy")
+datas += [('/home/jeffrey/.local/lib/python3.8/site-packages/astroquery/CITATION', 'astroquery')]
```

Comment:
I assume this has to change?

Reply:
Oh yeah, sorry I shall make that change.